sunbird_service_api_test/src/test/java/org/sunbird/common/action/CourseBatchUtil.java

```diff
@@ -11,6 +11,7 @@
 public class CourseBatchUtil {
 
   private static String openCourseBatchId = "";
+  private static String inviteOnlyCourseBatchId = "";
   public static final String TEST_NAME_CREATE_COURSE_BATCH_SUCCESS_OPEN_BATCH =
```

Comment:
Constants need to be named as TEST_{testName} -> currently _NAME seems to be appended as extra in most of the constants. Please check for all.